mssqlcli/mssql_cli.py

```diff
@@ -366,6 +368,20 @@ def execute_query(self, text):
             output = self._execute_interactive_command(text)
```

Comment:
I'm confused. You no longer call str(text) anywhere. Why is it not needed? Given removal in outer class I'm surprised we don't need to do anything here?

Reply:
I'm unsure how `text` could ever end up not having a string type, but to be sure I can throw in a conversion at the beginning of the function call for `execute_query`.